File: src/projectFolders/Programming/Programming/Java_Projects/withdraw.java

import java.util.Scanner;
class Scratch{
public static void main (String[] args){
String pin = "2003";
Scanner input = new Scanner (System.in);
for (int i = 0; i <= 2; i++){
System.out.println("Enter you pin:");
String enteredPin = input.nextLine();
if(enteredPin.equals(pin)) {
System.out.println("Login Successful!");
System.out.println("Options:");
System.out.println("Deposit enter 1");
System.out.println("Withdraw enter 2");
System.out.println("Chech balance enter 3");
int option = input.nextInt();
System.out.println("You entered option is: " + option);
break;
}else{
System.out.println("Login Unsuccessful!");
}
}
}
}

File: src/projectFolders/Programming/Programming/ C_projects/withdraw.c
#include<stdio.h>
int main(){
char pin [5] = "1234";
int attempt;
char enteredPin [5];
for(attempt = 0; attempt <=2; attempt ++){
printf("Enter your pin :");
scanf("%s", enteredPin);
if(strcmp (enteredPin, pin) == 0){
printf("Login successful \n");
break;
}else{
printf("Invalid");
}
}
}